Mathlib/Analysis/SpecialFunctions/CompareExp.lean

```diff
@@ -0,0 +1,229 @@
+/-
+Copyright (c) 2022 Yury Kudryashov. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Yury Kudryashov
+
+! This file was ported from Lean 3 source module analysis.special_functions.compare_exp
+! leanprover-community/mathlib commit 0b9eaaa7686280fad8cce467f5c3c57ee6ce77f8
+! Please do not edit these lines, except to modify the commit id
+! if you have ported upstream changes.
+-/
+import Mathlib.Analysis.SpecialFunctions.Pow.Asymptotics
+import Mathlib.Analysis.Asymptotics.AsymptoticEquivalent
+import Mathlib.Analysis.Asymptotics.SpecificAsymptotics
+
+/-!
+# Growth estimates on `x ^ y` for complex `x`, `y`
+
+Let `l` be a filter on `ℂ` such that `Complex.re` tends to infinity along `l` and `Complex.im z`
+grows at a subexponential rate compared to `Complex.re z`. Then
+
+- `Complex.is_o_log_abs_re_of_subexponential_im_re`: `Real.log ∘ Complex.abs` is `o`-small of
```

Comment:
This docstring is incorrect in Mathlib3. There is no function `is_o_log_abs_re_of_subexponential_im_re`. 

What to do here?

Reply:
You can change it here. Doc fixes are safe to do.
Fixing the mathlib3 docs is optional.

Reply:
Thanks!